driver/src/main/java/org/kaazing/k3po/driver/internal/netty/bootstrap/http/HttpChildChannelSource.java

```diff
@@ -125,9 +126,7 @@ public void channelClosed(ChannelHandlerContext ctx, ChannelStateEvent e) throws
             case CONTENT_COMPLETE:
                 break;
             default:
-                ChannelException exception = new ChannelException("Channel closed unexpectedly");
-                exception.fillInStackTrace();
-                fireExceptionCaught(httpChildChannel, exception);
+                Channels.fireChannelAborted(httpChildChannel);
```

Comment:
Good catch.  I'll added it for completeness.  Note, I believe it is impossible for that switch to ever be hit right now with a valid rpt.  And it's important to understand the limitation of the abort as implemented.  Specifically the enum is `REQUEST, CONTENT_CHUNKED, CONTENT_COMPLETE, UPGRADED `.  It is not possible to abort a request half way through, as that is the design of k3po.  I.E. you can't script an abort in the request as follows, because the http request is just configuration data that is flushed all to the wire at once.  Also, I don't believe chunking is supported yet, see https://github.com/k3po/k3po/issues/136, https://github.com/k3po/k3po/issues/11, and https://github.com/k3po/k3po/issues/233.

```
connect http://localhost:8080/path
connected
write method "GET"
abort
write header "Some" "header"

```

Thus I don't know how I'll test it.  Maybe the following:

```
connect http://localhost:8080/path
connected
abort

```